### ISSUE TITLE
aptly repo copy should accept a wildcard

### ISSUE THREAD

**Issue by aioue** [aptly-dev/aptly#269]
#### Use case

Copy all packages from one local repo to another
#### Expected behaviour

`aptly --dry-run=true repo copy foo-test foo-stable *` should show that it will copy all packages on local repo `foo-test` to local repo `foo-stable`
#### Actual behaviour

No packages are listed as being copied

**Comment by smira** [comment]
@aioue I agree that 'wildcard' should work, but right now "always true" condition is `Name` (which means Name is not empty)

**Comment by aioue** [comment]
Yes that's a good point. A simple workaround is `aptly --dry-run=true repo copy foo-test foo-stable Name *`.

**Comment by smira** [comment]
No, you don't need `*` at all, just `Name`
